client/src/app/containers/Game/CribAction.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import Button from '@material-ui/core/Button';
+import SendIcon from '@material-ui/icons/Send';
+
+import { Game } from './models';
+import { ActionInputProps } from './types';
+import { useGame } from './useGame';
+
+const expNumCardsToCribForGame = (game: Game) => {
+    const numPlayers = game.teams.reduce(
+        (num, team) => team.players.length + num,
+        0,
+    );
+    if (numPlayers > 2) {
+        return 1;
+    }
+    return 2;
+};
+
+const CribAction: React.FunctionComponent<ActionInputProps> = ({
+    isBlocking,
+}) => {
+    const {
+        clearSelectedCards,
+        game,
+        selectedCards,
+        submitBuildCribAction,
+    } = useGame();
+
+    const handleClick = async () => {
+        // TODO the submit build crib action should just take care of this probably
```

Comment:
:mag: what?

Reply:
Fun! I'm not sure what I meant! possibly that `clearSelectedCards` should be called by `submitBuildCribAction`, not this click handler.

Reply:
This is a shortcoming of how I've created and used `createActionHandler` in `useGame`. `createActionHandler` creates a rigidly defined function that each action has to do upon submission; in so doing, I've locked myself out of adding additional logic to only _some_ of the action handlers. I need to think more about how to do this better.